admin/src/utils/ita.js

```diff
@@ -149,7 +152,9 @@ const DESCRIPTOR_FIELDS = [
   "name",
   "description",
   "internal",
-  "source"
+  "source",
+  "deprecated",
```

Comment:
Where is "deprecated" being used now?

Reply:
This is added in the underlying branch. Parsing the config schema fails if this keyword is not added. Here's a comment in context: https://github.com/mozilla/hubs/pull/4030#pullrequestreview-625953237

Reply:
Nothing is actually using the "deprecated" flag yet though. I need to check if in-world menu/sprite colors are still using the old theme configs. If so, they're still important and should not be marked "deprecated" in a visible way until we change that.

Reply:
Update - I now hide the "deprecated" configs unless the `show_deprecated_configs` query string parameter is present.